.github/workflows/all_solutions.yml

```diff
@@ -626,7 +626,7 @@ jobs:
           path: C:\IntegrationTestWorkingDirectory\TestResults\**\*TestResults.xml
           if-no-files-found: error
 
-  create-package-rpm:
+  create-linux-packages:
```

Comment:
Is there a reason we are combining the building here?   I think it might be better to run each build in its own job so that that they run faster.  What are your thoughts?

Reply:
The reason is I wasn't sure if having 2 jobs running in parallels and creating artifacts in the same output directory `_build\CoreArtifacts` would cause a problem. I guess may be not, I can change that.

Reply:
It shouldn't since it acts like a share